### DISCUSSION TITLE
Group 5: Manual Scavenging in India

### DISCUSSION THREAD

**Discussion by bbdaniels** [gui2de/ppol768-spring23#161] [Group Projects Feedback]
Hi Group 5! You've chosen an excellent topic for your project -- your suspicion is right, however, and I am going to focus my feedback here on **narrowing down** the scope and scale of the project so that it is more tractable for the purposes of this course. I am speaking from experience as you know from my work in health in India -- it is always weird to propose something like a one-city or one-state study, but Indian cities are often larger than countries, so! I am much more concerned with your group developing a design in which the results are clear and intelligible than covering the whole country in any kind of sense. So let's stick to what you know and drill down to a project that is clear and concise.

For scope, I would recommend to pick one state and focus in from there. Your idea to focus on rural communities rather than urban centers is, I think, a good one -- since you will look at community-level interventions you want to make sure you have enough disparate communities to have a reasonable study design. While it is true that urban cores have many more people and there is a lot of interest, it will be very hard for you to make meaningful impact because of the structure of your problem. So I would stick with the narrower scale: While "rural India" is still enormous and diverse across states and regions, you can learn important lessons from working in one state that have a chance of making an impact elsewhere.

This brings me to the design and motivation. This is an intensive intervention! You rightly point out that the problem is mainly a failure of _governance_ in providing public goods, and you seem to be proposing interventions that nearly wholesale replace that shortcoming. Unfortunately, this (a) makes the project very very expensive; and (b) runs into the "poor people are poor" problem: your solution is to give people things they don't have. The reason this usually doesn't make for a strong research project is because the results are not that interesting: when you make people richer, their lives get better. What we want to use research for is, in general, to find out what is stopping people (or villages, or cities, or whatever) from doing better _with the resources they already have_.

Hence, there has to be a _market_ failure, in the sense that something is priced incorrectly. Our basic assumption is always that people are doing the best they can and that they know what is best for them. Therefore the only reason they could be reaching suboptimal outcomes given current resources is a mis-pricing (of inputs, outputs, labor, etc etc). If people are shoveling excrement instead of getting better jobs, they must have a good reason! Caste, unfortunately, is a very unique and hard-to-change reason, as it (presumably) implies that people doing these jobs (and jobs like them, such as animal carcass removal and tanning) cannot get paid fairly for other kinds of work. In that case, something like better community sanitation could be _bad_ for these people, as they may then have no work at all!

So you have two basic (intertwined) avenues here: A question about "how to improve sanitation systems", and a question about "how to then provide employment for peoples who have, through all of modern history, been effectively used as a human sanitation system". I would follow just one of those; and the latter is much more interesting. You could, for example, look at rural sanitation projects that have happened over the past decade, and try to figure out what happens to Dalits when sanitation _does_ improve -- do they get jobs? Do they not? Then, prospectively, you could try and suggest something to improve those outcomes: but this does not have to be an intervention project; doing just that with a thoughtful design would be a sufficient project!

In sum -- you are on the right track, but need to narrow down a lot so you can address one topic and question in detail in the next assignment. I hope this is helpful and look forward to seeing more from your group!

**Comment by BeaLeydier** [comment]
Hi team

Great research subject and great job providing context and motivation for it! I like it a lot. My main comments building off Ben's excellent suggestions, are these 

1. With your 3 research arms, you essentially have 3 studies in 1, and I would also recommend to just do one study in one. While it may be tempting to do very large scale studies with very different types of intervention to respond to big questions like "are demand-side or supply-side programs more effective to resolve this issue?" (and some studies do do that), it would be a much more ambitious study than what the scope of this class allows for (merely in terms of time), and it makes each individual intervention much harder to design because you would want to be able to say things comparatively about them, while they are precisely acting on different outcomes and constraints, so comparing them is not in itself trivial. All that to say, narrow it down!

1. My intuition is also that what you have as your arm 2 is the most interesting, and you can take it in both directions : what happens to Dalits from an economic perspective when sanitation improves; and what happens to sanitation/public defecation when Dalits get other jobs?

1. Small thing but it matters too - you will also need to improve the formatting for the next deliverables. Feel free to experiment editing the MD files in github directly so you can use the preview window and see how your formatting changes get rendered. Also try to harmonize the language - we don't really need the copy pasting of legalese defining open defecation at the beginning of your paper, and we want the tone (accessible, clear tone for a lay audience which you implement nicely in the intervention section) to be somewhat consistent throughout.

Cheers!

**Comment by bbdaniels** [comment]
# Concept Note Review

Hi team, this is an excellent expansion of the project along the lines we discussed in the project. The main revisions that are needed are to adapt the second half of the proposal into narrative text with appropriate exhibits (rather than the loooong series of bullet points as it is now). I think you have very well gotten hold of the concepts that we discussed in our meeting and I am excited to see this developed further. 

What you need now to do is the following. First, get an idea of the sample size. Do not say "based on exploratory data analysis we will..." because you need the complete proposal now! How many communities are able to be included in your RCT? Which/how many are being targeted by the desludging program (it is OK if you do not know this -- but after next week you should consider how you would develop a power analysis to understand what would be your detectable effect sizes for a given number of such sites). In mechanisms, you should specify whether you are doing community-level outcomes or individual-level outcomes, and how you will measure them.

In the RCT section, you need to expand very carefully on how you are conducting randomization and project delivery (how are you getting participants? what counts as makeup? are these community or individual randomization?). Then, set out how many participants you will want in each arm, including level or randomization and relevant stratification. This will give you the basis for firming up your exact experimental comparison, which I think you are very close to finalizing.

Finally, be very clear about the data sources here. Who are you "surveying" about this? How many people are you surveying? What if they don't respond? Then, ultimately, which groups are you comparing for which outcomes? Having too many outcomes also affect statistical power -- you can investigate Bonferroni or Benjamini-Hochberg approaches to dealing with this as you move forward into figuring out your detectable effect sizes. I see that you have begun to deal with these in the "Concerns" section -- either build this out or integrate it better into your design. Great work so far!
